string frameID
string distortion_model
int32 height
int32 width
int32 binning_x
int32 binning_y
